typescript/sdk/src/deploy/HyperlaneAppChecker.ts

```diff
@@ -122,17 +143,24 @@ export abstract class HyperlaneAppChecker<
     }
   }
 
-  async checkOwnership(
-    chain: ChainName,
-    owner: types.Address,
-    ownables: Ownable[],
-  ): Promise<void> {
-    await Promise.all(
-      ownables.map(async (contract) => {
+  // TODO: Require owner in config if ownables is non-empty
+  async checkOwnership(chain: ChainName, owner: types.Address): Promise<void> {
+    const isOwnable = (_: string, contract: any): contract is Ownable => {
+      return (
+        contract !== null &&
+        typeof contract === 'object' &&
+        contract.owner &&
+        contract.transferOwnership
+      );
+    };
```

Comment:
still think we shouldn't be relying on types here

Reply:
Agreed, leaving as a follow-up for expediency so I don't need to make getFilteredFlattendContracts take an async function

Reply:
Will address in #2042